### ISSUE TITLE
Remove redirect sites

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/blocked-org-uk#99]
Can we remove "redirect sites" from browsable the dataset?  Is there a way to identify them eg by "redirect permanent" messages, for instance?

**Comment by dantheta** [comment]
I think it's possible with the way the data is stored, but we'd need to refine the criteria a bit, since a redirect URL can be blocked by ISP filters independently of the site it refers to.    For instance, http://13ghosts.warnerbros.com.  This URL is blocked on several networks, but the page it redirects to is not.

When a URL redirects to a blocked site, the result for that URL is "status = blocked".  This might be something that we want to change (the last discussion on it was in 2015 or so), but it would require reprocessing a lot of results.  The stored results don't contain the number of redirects that took place before arriving at the ISP parental control/malware/copyright warning page.

**Comment by JimKillock** [comment]
*Any* redirect is a legitimate block for an ISP filter I suspect. Their reasoning AIUI is that redirects can be easily / arbitrarily / frequently moved to more dubious sites. 

We don't need to worry about redirects that are not blocked. However, there is little point in our users reporting redirects to the ISPs (they will simply say "we filter them"). So I would exclude them from search by default, as we do with sites that appear to be porn or escort agencies.

**Comment by JimKillock** [comment]
I think after observing what goes on with the ISPs, we can see that:
* They tend to block redirects;
* They will generally unblock legitimate redirects

Thus the ideal answer may be to
* Record and mark sites as redirects
* Allow searches to exclude or include redirects

**Comment by JimKillock** [comment]
Should we close this?

**Comment by JimKillock** [comment]
@dantheta Should we close this? It seems like this is at a "wontfix" point.

**Comment by dantheta** [comment]
I think so.  We've got the data to approach it again if needed.
